sllurp/llrp.py

```diff
@@ -853,6 +861,12 @@ def send_SET_READER_CONFIG(self, onCompletion):
                 },
             }
         }
+        keepalive_spec = {
+         'KeepaliveTriggerType': 1,
+         'TimeInterval': llrp_keepalive_interval_ms
+        }
+        if(llrp_keepalive_interval_ms != 0):
+            msg['SET_READER_CONFIG']['KeepaliveSpec'] = keepalive_spec
```

Comment:
You can create and assign directly the keepalive_spec block in one time inside the if.
So that, you don't create the dict for nothing if not to be used.